python/ray/_private/thirdparty/pynvml/__init__.py

```diff
@@ -0,0 +1 @@
+from ray._private.thirdparty.pynvml.pynvml import *
```

Comment:
Just a suggestion: It'd be good to keep track of which version of `nvidia-ml-py` is being imported (for future references) as metadata. `pynvml` does not have any sort of `__verion__` strings.

Reply:
Also, I think this is fine with the current space, but just heads up: As in https://github.com/wookayin/gpustat/issues/153, this does not re-export private members starting with `_`. Private APIs of `pynvml` are not being used so it'd be fine :)

Reply:
+1. We should track the version of `nvidia-ml-py` that we are vendoring here.

Reply:
sounds good for the version.

@wookayin for private APIs, yes for now I dont think we will use it so keeping it that way + change in case we need later